.travis.yml

```diff
@@ -34,16 +34,10 @@ env:
     - TEST_AIIDA_BACKEND=django COMPILE_DOCS=true
 
 install:
-    # This is required for some recent features of setup.py
+    # Upgrade pip setuptools and wheel to be able to run the next command
     - pip install -U pip wheel setuptools
```

Comment:
We could in principle, but I guess this leads to a redundancy if we keep pip, wheel, and setuptools aldo in  `setup_requirements.py` 
No, unfortunately I have no "magic" script to discover what is the newest version of a package. Also because this is not per se useful, as one has to discover what is the latest version of a package that does not break the tests. So far, I tested manually each dependency upgrade.